src/manifest.json

```diff
@@ -2,7 +2,7 @@
   "name": "LiveTL - Translation Filter for Streams",
   "options_ui": {
     "page": "options.html",
-    "open_in_tab": true
+    "open_in_tab": false
```

Comment:
does this need to be set to false? would not want to change existing behaviour

Reply:
This allows an Options Tab in the Manage Extensions page in Firefox, or just gives a simple pop-up in Chrome.

Its just more cleaner imo, and doesn't change the overall behavior of the extension.

Reply:
Oh wow I actually did not know that is what that does. That's great! Thanks for adding this.